ISISScientificComputing/autoreduce
# Remove raw stomp use in post process admin

### Summary of work
* Remove raw stomp use in `post_process_admin.py`
* Change tests to use QueueClient in mocks
* This also clears up the issue from #518 which caused the subscription in the `autoreduction_queue_processor` to fail.

### How to test your work
* Ensure unit tests pass
* Code is currently on development node (might be worth checking before running) so it should be possible to resubmit a run and if it completes with no issue then this passes

Part of #517
